go.mod

```diff
@@ -37,6 +37,8 @@ require (
 )
 
 replace (
+	github.com/projectcalico/api => github.com/coutinhop/api v0.0.0-20211029224736-72ed23299ba3
+	github.com/projectcalico/libcalico-go => github.com/coutinhop/libcalico-go v1.7.2-0.20211029225451-23efd56017ad
```

Comment:
Appears to be dependent on https://github.com/projectcalico/api/pull/40 and https://github.com/projectcalico/libcalico-go/pull/1539